app/assets/javascripts/application.js

```diff
@@ -40,8 +40,16 @@
 //= require controllers/ops/log_collection_form_controller
 //= require controllers/ops/tenant_form_controller
 //= require controllers/ops/tenant_quota_form_controller
+//= require controllers/container_dashboard/util/util-module
+//= require controllers/container_dashboard/util/dashboard-utils
+//= require controllers/container_dashboard/util/chart-data-mixin
+//= require controllers/container_dashboard/util/chart-config-util
 //= require controllers/container_dashboard/container_dashboard_controller
-//= require controllers/container_dashboard/container_dashboard_utils
+//= require controllers/container_dashboard/card/card.module
+//= require controllers/container_dashboard/card/heatmaps/heatmaps-card-directive
+//= require controllers/container_dashboard/charts/charts-module
+//= require controllers/container_dashboard/charts/heatmap/heatmap-directive
+//= require controllers/container_dashboard/charts/heatmap/heatmap-legend-directive
```

Comment:
@himdel So in the long run these are meant to be shared across the project but throwing them all in the directives directory seemed a bit unorganized. Where should these sit?

Also, I can use one of these instead of the test.html\haml from the high_voltage path